### PR TITLE
Add support for EventStoreDB

### PR DESCRIPTION
**Pull request by morrislaptop**
+ some small fixes in the test class

### CODE REVIEW

File: tests/Feature/BaseServiceTest.php [hunk 1]
```diff
@@ -42,10 +42,10 @@ function it_enables_services()
             $mock->shouldReceive('imageIsDownloaded')->andReturn(true);
 
             // This is the actual assertion
-            $mock->shouldReceive('bootContainer')->with(['getmeili/meilisearch']);
+            $mock->shouldReceive('bootContainer')->once();
```
Comment (mattstauffer) [inline tests/Feature/BaseServiceTest.php:45]: Why this replacement?

Comment (morrislaptop) [reply tests/Feature/BaseServiceTest.php:45]: Without specifying a call count the default is zero or more - http://docs.mockery.io/en/latest/reference/expectations.html#declaring-call-count-expectations

Tests were currently passing no matter the value of `bootContainer` or `getmeili/meilisearch`

Comment (mattstauffer) [reply tests/Feature/BaseServiceTest.php:45]: HA! Thanks @morrislaptop! Could you *keep* the original `with`?

Comment (morrislaptop) [reply tests/Feature/BaseServiceTest.php:45]: Hi @mattstauffer the original with isn't correct, it's going to have to be something like

```php
$mock->shouldReceive('bootContainer')->with('-p "${:port}":7700 \
            -v "${:volume}":/data.ms \
            "${:organization}"/"${:image_name}":"${:tag}"', [
                "organization" => "getmeili",
                "image_name" => "meilisearch",
                "port" => 7700,
                "tag" => "v1.1.1",
                "volume" => null,
                "container_name" => "TO--meilisearch--v1.1.1",
            ])->once();
```

which doesn't look very nice, any suggestions?

Comment (mattstauffer) [reply tests/Feature/BaseServiceTest.php]: Eh, let's just keep it as you have it for now and I'll dig in later to see if we really need that. Thanks so much!

**Comment by mattstauffer** [comment]
@morrislaptop would you mind modifying this PR to allow me to make edits? I made some changes locally I'd like to push up; I'd like to push the changes up and then ask you to take a look at the failing tests.

Thanks!

**Comment by morrislaptop** [comment]
@mattstauffer it was already checked but i've unchecked and checked it, does that work?

**Comment by mattstauffer** [comment]
@morrislaptop Hm, nope, not sure why it isn't. Let me just throw you a patch file quickly.

**Comment by mattstauffer** [comment]
Uh... I merged main in and pushed, which should've done almost nothing, and instead it's.. doing this? I'm not sure what's going on. :/

**Comment by morrislaptop** [comment]
lol! Give me 5, I'll merge master in, there was only conflict on README IIRC

**Comment by mattstauffer** [comment]
Thanks @morrislaptop! OK, so now where we are right now, we have two tests failures; would you mind taking a look?

image

**Comment by morrislaptop** [comment]
@mattstauffer Tests and style fixed 👍

**Comment by mattstauffer** [comment]
Thanks @morrislaptop!

**Pull request merged**